packages/app/src/redux/slices/editingProject.ts

```diff
@@ -61,6 +61,7 @@ export const editingProjectSlice = createSlice({
       cycleLimit: BigNumber.from(0),
       configured: BigNumber.from(0),
       ballot: constants.AddressZero,
+      basedOn: BigNumber.from(0),
```

Comment:
replace `previous` with `basedOn` in the object instead of adding a property at the end.